src/anders_mathiesen_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+import random
+
+
+class Walker:
+    """
+     Simulate a student’s way home to Pentagon after a hard night at Samfunnet
+    """
+    def __init__(self, x0, home):
+        self.home = home
+        self.x = x0
+        self.x0 = x0
+        self.steps = 0
+
+    def move(self):
+        """
+        a method move to take one step
+        """
+        self.steps += 1
+        if random.randint(0, 1):
+            self.x += 1
+        else:
+            self.x -= 1
+
+    def is_at_home(self):
+        """
+        a method is_at_home to check whether the student is at home
+        :return:
+        boolean
+        true if you are home else false
+        """
+        if self.x == self.home:
+            return True
+        return False
+
+    def get_position(self):
+        """
+        a method get_position to access the students current position
+        :return:
+        int
+        true if you are home else false
+        """
+        return self.x
+
+    def get_steps(self):
+        """
+        a method get_steps to access the number of steps the student has taken in total
+        :return:
+        int
+        steps take to go home
+        """
+        return self.steps
+
+    def reset(self):
+        """
+        a method get_steps to access the number of steps the student has taken in total
+        :return:
+        int
+        steps take to go home
+        """
+        self.x = self.x0
+        self.steps = 0
+
+    def go_home(self):
+        """
+        simulate a walk home
+        """
+        while not self.is_at_home():
+            self.move()
+
+
+class Simulation:
+    """
+     Simulate a person way home
+    """
+    def __init__(self, start, home, seed):
+        self.list_of_position = []
+        self.start = start
+        self.home = home
+        random.seed(seed)
+
+    def single_walk(self):
+        """
+       Simulate single walk from start to home, returning number of steps.
+
+       Returns
+       -------
+       int
+           The number of steps taken
+       """
+        steps_take = 0
+        position = self.start
+        while position != self.home:
+            if random.randint(0, 1):
+                position += 1
+            else:
+                position -= 1
+            steps_take += 1
+            self.list_of_position.append(position)
+        return steps_take
```

Comment:
You are supposed to use a `Walker` instance here. Not redo all the logic in the `Walker` class.